signed/verify_test.go

```diff
@@ -132,7 +132,7 @@ func Test(t *testing.T) {
 			role: "root",
 			name: "expired",
 			exp:  &expiredTime,
-			err:  ErrExpired{"root", expiredTime.Format("2006-01-02 15:04:05 MST")},
+			err:  ErrExpired{"root", expiredTime.Format("Mon Jan 2 15:04:05 MST 2006")},
```

Comment:
Which standard is this supposed to be using and can we just use a constant defined in the `time` package. Looks almost identical to `time.UnixDate`